JPF/PubAdvWebScrape.ipynb

```diff
@@ -0,0 +1,605 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 70,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2016-11-26T22:41:18.806478",
+     "start_time": "2016-11-26T22:41:12.447122"
+    },
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>ADDR</th>\n",
+       "      <th>LANDLORD</th>\n",
+       "      <th>HPDV</th>\n",
+       "      <th>DOBV</th>\n",
```

Comment:
those are the data that the Pub Advocate site provides. 
the Address of the Bad Landlord Building
the name of the Bad Landlord
the #of HPD Violations the Public Adv has assigned, based on their method, to the BUILDING
the #of DOB Violations the Public Adv has assigned, based on their method, to the BUILDING
see: http://landlordwatchlist.com/search.html
see: http://advocate.nyc.gov/Methodology